src/blog/2020-10-14-tagged-unions.md

```diff
@@ -7,7 +7,7 @@ description: "Tagged Unions in JavaScript and TypeScript"
 
 ## Why Should I Care?
 
-Redux. MobX. XState. Vuex. RxJS. State management is hard, and developers are always looking for a tool to help them. Tagged unions are a programming pattern that you can use with immutable state libraries, or even by itself. Tagged unions make it simple to visualize all the states your application can be in, and make it difficult to access the wrong data at the wrong time.
+Redux. MobX. XState. Vuex. RxJS. State management is hard, and developers are always looking for a tool to help them. Tagged unions are a programming pattern that you can use with immutable state libraries, or even on their own. Tagged unions make it simple to visualize all the states your application can be in, and prevent you from accessing the wrong data at the wrong time.
 
 Note: Tagged unions are also called "algebraic data types" or "enums" in different programming languages. I'm going to use the term "tagged unions" because I think it's easier to understand.
```

Comment:
It might be obvious to you, but the name "tagged unions" doesn't evoke any meaning to me and it's not clear why they are called that. I feel like a sentence to explain why they are called that might be nice.